src/geometry/sphere.cc

```diff
@@ -0,0 +1,29 @@
+#include "sphere.h"
+#include "ray.h"
+#include <iostream>
+
+Sphere::Sphere(Vertex position, float radius, ColorDbl color)
+    : SceneObject(position), color_(color) {
+  assert(radius > 0);
+  radius_ = radius;
```

Comment:
Why not initialize this like above and do the assert on radius_? Makes no difference though